src/Lock.vue

```diff
@@ -148,7 +148,7 @@
       let originalFocusedElement;
 
       onMounted(() => {
-        data.value.vue = getCurrentInstance();
+        data.value.vue = (getCurrentInstance() || {}).proxy;
```

Comment:
technically we don't need "real" instance of the component hidden behind non fancy name `proxy`. One can use `getCurrentInstance.uid`

But I am a little concerned about `|| {}` construct - it is possible that there is no current instance? Logic will be broken in this case.

Thinking would the following work
```tsx
data.value.vue = getCurrentInstance().uid;
```

Reply:
Ok, you were right. There are all different types in 2.6, 2.7 and 3 and proxy is the only one in common